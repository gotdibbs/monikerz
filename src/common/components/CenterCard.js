import React from 'react';

export default function CenterCard({ children, className }) {
    return (
        <article className={['card-center', className].join(' ')}>
            { children }
        </article>
    );
}