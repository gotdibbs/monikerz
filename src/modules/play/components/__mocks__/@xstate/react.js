let state = {
    matches: jest.fn()
};

const send = jest.fn();

export const useMachine = () => {
    return [useMachine.state, useMachine.send];
};

useMachine.state = state;
useMachine.send = send;