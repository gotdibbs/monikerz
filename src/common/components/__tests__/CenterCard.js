import React from 'react';
import { render } from '@testing-library/react';
import CenterCard from '../CenterCard';

describe('CenterCard', () => {
    it('should render', () => {
        const { asFragment } = render(<CenterCard />);
        expect(asFragment()).toMatchSnapshot();
    });
});