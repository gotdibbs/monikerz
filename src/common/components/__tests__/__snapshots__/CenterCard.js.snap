// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterCard should render 1`] = `
<DocumentFragment>
  <article
    class="card-center "
  />
</DocumentFragment>
`;
